src/components/Persona/Persona.scss

```diff
@@ -23,6 +23,19 @@ $ms-color-initials-orange:      #da532c;
 $ms-color-initials-red:         #ee1111;
 $ms-color-initials-darkRed:     #b91d47;
 
+// Persona sizes
+$tinyPersonaSize: 30px;
```

Comment:
Typically these kinds of variables take the form $ComponentProperty-value, e.g. $PersonaSize-tiny (see SearchBox's variables for an example). I realize that this convention is not documented--I'll fix that soon--but in the meantime, could you patch these up to conform to that pattern? Thanks!